xml/art_sle_amazon_ecs.xml

```diff
@@ -132,6 +139,69 @@
           cluster. Remember to select the ECS role when launching the
           instance.
         </para>
+        <para>
+          You may use the Amazon EC2 Web Console to launch instances.
+          The <productname>&sls</productname> ECS images are found in the
+          so called General Catalog. From the
+          <emphasis role="strong">Instances</emphasis> view select
+          <guibutton>Launch Instance</guibutton>. In the left
+          column select the <guimenuitem>Community AMIs</guimenuitem>
+          tab. Check the <guibutton>SUSE Linux</guibutton> box
+          on the left and enter <emphasis role="strong">ecs</emphasis> into
+          the search box. When configuring the instance parameters in the
+          <emphasis role="strong">Configure Instance Details</emphasis> view
+          select the <guimenuitem>ecsInstanceRole</guimenuitem> as
+          the IAM role. Then launch the instance.
+        </para>
+        <para>
+          Alternatively you can use the <command>aws</command> command line
+          tool to run instances, see
+          <link xlink:href="http://docs.aws.amazon.com/cli/latest/reference/ec2/run-instances.html">run-instances</link> for details.
+        </para>
+        <screen>$ aws ec2 run-instances --image-id <replaceable>IMAGE_ID_SEE_PINT_OUTPUT</replaceable> --region <replaceable>EC2_REGION_NAME</replaceable> --key-name <replaceable>SSH_KEY_PAIR_NAME</replaceable> --instance-type <replaceable>DESIRED_INSTANCE_TYPE</replaceable> --subnet-id <replaceable>SUBNET_ID</replaceable> --iam-instance-profile 'Arn=<replaceable>PROFILE_PATH</replaceable>'</screen>
```

Comment:
Technically it's absolutely correct. However, IMHO it would be better to introduce some linebreaks to improve readability, for example:

```
<screen>$ aws ec2 run-instances --image-id <replaceable>IMAGE_ID_SEE_PINT_OUTPUT</replaceable> \
    --region <replaceable>EC2_REGION_NAME</replaceable> \
  [...]</screen>
```